Sequence.h

```diff
@@ -17,11 +17,14 @@ class Sequence {
     void begin_sequence();
     void abort_seq();
     void set_ready();
+    void setLed(int led_val);
```

Comment:
not sure why setLed is here - you can do it via the LightTree reference.

Reply:
I  wrote a test sequence.cpp , and did cut and past when was working. Then slavishly copied the setLed, gateAct, and playTone functions too. Agreed can get rid of them